src/pg_probackup.h

```diff
@@ -677,8 +677,8 @@ typedef struct BackupPageHeader
 typedef struct BackupPageHeader2
 {
 	XLogRecPtr  lsn;
-	int32	    block;			 /* block number */
-	int32       pos;             /* position in backup file */
+	int64	    block;			 /* block number */
+	int64       pos;             /* position in backup file */
```

Comment:
Change to this struct is breaking change. One could not then easily restore backups made before this change. Therefore, patch have to be much more complex.